### PR TITLE
4354 add object id to cr

### PR DESCRIPTION
**Pull request by wschaeferB**

### CODE REVIEW

File: Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/SqliteEamDb.java [hunk 1]
```diff
@@ -1156,4 +1170,25 @@ private void releaseSharedLock() {
         rwLock.readLock().unlock();
     }
 
+    @Override
+    boolean doesColumnExist(Connection conn, String tableName, String columnName) throws SQLException {
+        final String tableInfoQueryTemplate = "PRAGMA table_info(%s)";  //NON-NLS
+        ResultSet resultSet;
+        Statement statement = conn.createStatement();
+        boolean columnExists = false;
+        resultSet = statement.executeQuery(String.format(tableInfoQueryTemplate, tableName));
```
Comment (APriestman) [inline Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/SqliteEamDb.java]: I think this should be in the try-catch block to make sure everything gets closed

File: Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/PostgresEamDb.java [hunk 2]
```diff
@@ -189,36 +191,57 @@ protected Connection connect() throws EamDbException {
...
         }
     }
 
+    @Override
+    boolean doesColumnExist(Connection conn, String tableName, String columnName) throws SQLException {
+        final String objectIdColumnExistsTemplate = "SELECT EXISTS (SELECT 1 FROM information_schema.columns WHERE table_name='%s' AND column_name='%s')";  //NON-NLS
+        ResultSet resultSet;
+        Statement statement = conn.createStatement();
+        boolean columnExists = false;
+        resultSet = statement.executeQuery(String.format(objectIdColumnExistsTemplate, tableName, columnName));
```
Comment (APriestman) [inline Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/PostgresEamDb.java]: Same as the other place - this should probably be in the try-catch block to close everything.

File: Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/AbstractSqlEamDb.java [hunk 3]
```diff
@@ -808,9 +808,9 @@ public void addArtifactInstance(CorrelationAttributeInstance eamArtifact) throws
         String sql
                 = "INSERT INTO "
                 + tableName
-                + "(case_id, data_source_id, value, file_path, known_status, comment) "
+                + "(case_id, data_source_id, value, file_path, known_status, comment, object_id) "
```
Comment (rcordovano) [inline Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/AbstractSqlEamDb.java]: Let's use src_file_obj_id fr the column name.

Comment (rcordovano) [reply Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/AbstractSqlEamDb.java]: Actually, per Brian Carrier, let's use file_obj_id.

**Pull request merged**